microcosm_flask/conventions/crud.py

```diff
@@ -153,10 +154,14 @@ def configure_retrieve(self, ns, definition):
         :param definition: the endpoint definition
 
         """
+        request_schema = definition.request_schema or Schema()
+
         @self.graph.route(ns.instance_path, Operation.Retrieve, ns)
+        @qs(request_schema)
```

Comment:
I'm pretty sure it's a noop, I also intentionally added the tests to an existing route and it retrieve seems to return the expected values (both for success and failure). Anything else I should test?